airbyte-integrations/connectors/source-jina-ai/metadata.yaml

```
@@ -0,0 +1,40 @@
+data:
+  allowedHosts:
+    hosts:
+      - "https://*.jina.ai"
```

I wonder if we should just list these two endpoints explicitly.

```suggestion
    hosts:
      - "https://s.jina.ai"
      - "https://r.jina.ai"
```

---

:white_check_mark: